src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -814,6 +829,11 @@ private void InheritNullableStateOfTrackableType(int targetSlot, int valueSlot,
             }
         }
 
+        private TypeSymbol GetSlotType(int slot)
+        {
+            return variableBySlot[slot].Symbol.GetTypeOrReturnType().TypeSymbol;
```

Comment:
>GetTypeOrReturnType [](start = 47, length = 19)

It looks like we also have ```VariableType``` helper in the base class. Would it be more appropriate to use it here?